packages/cf-style-const/src/variables.js

```diff
@@ -95,5 +95,7 @@ export default {
   colorPurpleDark: darken(colorOffsetDark, '#d91698'),
   colorTwitterBlue: '#00aced',
   colorFacebookBlue: '#3b5998',
-  colorMainBackground: '#e6e6e6'
+  colorMainBackground: '#e6e6e6',
+  zIndexMax: 1000,
+  modalBoxShadow: '0 0 0 9999px rgba(0, 0, 0, .7)'
```

Comment:
That box-shadow looks like a soft drop shadow, where as the appstore modal is using this giant shadow to implement a page fade.

<img width="751" alt="screen shot 2017-06-02 at 5 01 00 pm" src="https://cloud.githubusercontent.com/assets/1265379/26746521/989ff992-47b5-11e7-91fe-8465b760d31d.png">